xds/src/main/java/io/grpc/xds/XdsLoadBalancer.java

```diff
@@ -91,17 +88,23 @@ public void handleResolvedAddressGroups(
       List<EquivalentAddressGroup> servers, Attributes attributes) {
     Map<String, ?> newRawLbConfig = checkNotNull(
         attributes.get(ATTR_LOAD_BALANCING_CONFIG), "ATTR_LOAD_BALANCING_CONFIG not available");
-    LbConfig newLbConfig = ServiceConfigUtil.unwrapLoadBalancingConfig(newRawLbConfig);
-    fallbackPolicy = selectFallbackPolicy(newLbConfig, lbRegistry);
+
+    ConfigOrError<XdsConfig> cfg =
+        XdsLoadBalancerProvider.parseLoadBalancingConfigPolicy(newRawLbConfig, lbRegistry);
+    if (cfg.getError() != null) {
+      throw cfg.getError().asRuntimeException();
+    }
```

Comment:
Do you expect all balancers to have this check-and-throw? If so, maybe we should add a `getConfigOrThrow()` to `ConfigOrError`.